src/command_modules/azure-cli-netappfiles/azure/cli/command_modules/netappfiles/_help.py

```diff
@@ -0,0 +1,466 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+# pylint: disable=line-too-long
+
+
+helps['netappfiles'] = """
+    type: group
+    short-summary: Manage Azure NetApp Files (ANF) Resources.
+"""
+
+# account
+
+helps['netappfiles account'] = """
+    type: group
+    short-summary: Manage Azure NetApp Files (ANF) Account Resources.
+"""
+
+helps['netappfiles account create'] = """
+    type: command
+    short-summary: Create a new Azure NetApp Files (ANF) account. Note that active directory can only be applied to an existing account (using set/update).
+    parameters:
+        - name: --account-name -a -n
+          type: string
+          short-summary: The name of the ANF account
+        - name: --tags
+          type: string
+          short-summary: A list of space separated tags to apply to the account
+    examples:
+        - name: Create an ANF account
+          text: >
+            az netappfiles account create -g group --account-name name -l location
+"""
+
+helps['netappfiles account set'] = """
+    type: command
+    short-summary: Sets the tags or the active directory details for a specified ANF account. Sets the active directory property to exactly what is provided. If none is provided then the active directory is removed, i.e. provide empty [].
+    parameters:
+        - name: --account-name -a -n
+          type: string
+          short-summary: The name of the ANF account
+        - name: --tags
+          type: string
+          short-summary: A list of space separated tags to apply to the account
+        - name: --active-directories
+          type: string
+          short-summary: An array of active directory (AD) settings in json format. Limitation one AD/subscription. Consists of the fields username (Username of Active Directory domain administrator), password (Plain text password of Active Directory domain administrator), domain (Name of the Active Directory domain), dns (Comma separated list of DNS server IP addresses for the Active Directory domain), smb_server_name (NetBIOS name of the SMB server. This name will be registered as a computer account in the AD and used to mount volumes. Must be 10 characters or less), organizational_unit (The Organizational Unit (OU) within the Windows Active Directory)
```

Comment:
For this alone, this module can only be created in preview.

Reply:
> For this alone, this module can only be created in preview.

Sorry I don't really get your meaning here. Can you explain a bit further?